paddle/fluid/platform/gpu_info.cc

```diff
@@ -476,5 +472,13 @@ bool IsCudaMallocRecorded(int dev_id) {
   return RecordedCudaMallocHelper::Instance(dev_id)->NeedRecord();
 }
 
+void UpdateGpuMemStat(int mode, int dev_id, size_t size) {
```

Comment:
Bad code smell of the use of `mode` for arithmetic method selection. I prefer the original `STAT_INT_ADD` statement to the use of `mode` parameter if no better way could be found.

Reply:
done